pyQuARC/code/custom_validator.py

```diff
@@ -119,6 +119,26 @@ def dif_standard_product_check(*field_values):
                     break
         return {"valid": validity, "value": value}
 
+    @staticmethod
+    def standard_product_check(*field_values):
+        """
```

Comment:
Why rename the method? What's the benefit of using one or the other method name? Can the older model name not be used?

Reply:
we already have a `dif_standard_product_check` method, no?

Reply:
Yes, we have `dif_standard_product_check`  that is used to check the extended metadata for DIF schema, but the `standard_product_check`  is used to check the `StandardProduct` key inside the collection for schema echo-c and umm-c. 
I thought, creating a new function solved the problem, but it shows the error again and now I changed the code which is different from the previous function `dif_standard_product_check` .

Reply:
If it's just checking values in a constrained list, don't we already have a check that does this? why start a new one?

Reply:
it's not that simple, some of these "fields" are not regular string fields, they are a list and you gotta check each of the item in the list to see if it exists. i believe this is one such case. similarly, and formatting is different for dif and other types, hence I think a new check is needed.

Reply:
Thank you @slesaad didi. Worked on that.

Reply:
Did you test it? Everything working well?

Reply:
I tested it on the "UMM-C record (C1577484501-LARC_ASDC)".  Yes, everything working well.